### PR TITLE
Task/bin249 implement services content on section

### PR DESCRIPTION
**Pull request by GustavoUT22**
#  Implement Service Content in Service Section 

## Summary

I implemented service.data.ts to handle data integration, added and styled the ServiceContent component, and adjusted Services.tsx to integrate it. I also moved the interfaces to a separate file for better organization.

### Problem

The service Content wasn't implemented

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor
- [ X] Styling
- [ ] Testing
- [ ] Chore
- [ ] Other: ...

## Checklist:

These items must be completed before the code review

- [ ] Check that branch is pointing to production environment
- [X ] Remove console.logs
- [ X] Check imports format
- [ X] Check variables destructure
- [ X] Check docs
- [ X] Review the entire pull request yourself before sending it to the reviewer
- [ ] Check task requirements on Monday or Sprint Planning sheet

### CODE REVIEW

**Review by CrhistianT7 — APPROVED** [review]
LGTM!!!

File: src/components/Services/Services.data.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import { ServicesContentProps } from 'types/components/serviceSection'
+import serviceImg from './../../assets/images/service-img.png'
+
+export const serviceContentData: ServicesContentProps = {
+  'grafic-pieces': {
+    id: 'grafic-pieces',
+    title: 'home.services.design.development.of.graphic.pieces',
+    description:
+      'home.services.design.development.of.graphic.pieces.description',
+    backlinks: [
```
Comment (mardecera) [inline src/components/Services/Services.data.ts:10]: Not backlinks! Includes!

File: src/components/Services/ServicesContent/ServicesContent.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <div className="left-content">
+        <div className="what-is">
+          <H4 $weight="bold">{intl('home.services.content.title.what.is')}</H4>
+          <p>{intl(props.description)}</p>
+        </div>
+        <div className="what-includes">
+          <H4 $weight="bold">
+            {intl('home.services.content.title.what.includes')}
+          </H4>
+          <ul className="backlinks-wrapper">
```
Comment (mardecera) [inline src/components/Services/ServicesContent/ServicesContent.tsx:22]: Not backlinks

**Review by mardecera — CHANGES_REQUESTED** [review]
pr-gustavo

In tablet and mobile, remains after clicking it.

File: src/components/Services/ServicesContent/ServicesContent.tsx (hunk 2 above)
Comment (GustavoUT22) [reply src/components/Services/ServicesContent/ServicesContent.tsx:22]: done!

File: src/components/Services/Services.data.ts (hunk 1 above)
Comment (GustavoUT22) [reply src/components/Services/Services.data.ts:10]: done!

**Pull request merged**